Library/Formula/binwalk.rb

```diff
@@ -15,57 +13,61 @@ class Binwalk < Formula
+  depends_on "pyside"
   depends_on :python if MacOS.version <= :snow_leopard
-  depends_on 'p7zip'
-  depends_on 'ssdeep'
-  depends_on 'xz'
+  depends_on "p7zip"
+  depends_on "ssdeep"
+  depends_on "xz"
 
-  resource 'pyqtgraph' do
-    url 'http://www.pyqtgraph.org/downloads/pyqtgraph-0.9.8.tar.gz'
-    sha1 '4ea6891f562c09ff13555ccb33fe05c315a70cf1'
+  resource "pyqtgraph" do
+    url "http://www.pyqtgraph.org/downloads/pyqtgraph-0.9.10.tar.gz"
+    sha256 "4c0589774e3c8b0c374931397cf6356b9cc99a790215d1917bb7f015c6f0729a"
   end
 
-  resource 'numpy' do
-    url "http://downloads.sourceforge.net/project/numpy/NumPy/1.8.1/numpy-1.8.1.tar.gz"
-    sha1 "8fe1d5f36bab3f1669520b4c7d8ab59a21a984da"
+  resource "numpy" do
+    url "http://downloads.sourceforge.net/project/numpy/NumPy/1.9.2/numpy-1.9.2.tar.gz"
+    sha256 "325e5f2b0b434ecb6e6882c7e1034cc6cdde3eeeea87dbc482575199a6aeef2a"
   end
 
-  resource 'scipy' do
-    url "http://downloads.sourceforge.net/project/scipy/scipy/0.14.0/scipy-0.14.0.tar.gz"
-    sha1 "faf16ddf307eb45ead62a92ffadc5288a710feb8"
+  resource "scipy" do
+    url "http://downloads.sourceforge.net/project/scipy/scipy/0.15.1/scipy-0.15.1.tar.gz"
+    sha256 "a212cbc3b79e9a563aa45fc5c517b3499198bd7eb7e7be1e047568a5f48c259a"
+  end
+
+  resource "capstone" do
+    url "https://pypi.python.org/packages/source/c/capstone/capstone-3.0.2.tar.gz"
+    sha256 "b32022fe956e940f8e67c17841dd3f6f1c50a60e451f9b5ce1f4dd2e5c5b3339"
   end
 
   def install
-    ENV.prepend_create_path 'PYTHONPATH', libexec+'lib/python2.7/site-packages'
-    numpy_args = [ "build", "--fcompiler=gnu95",
-                   "install", "--prefix=#{libexec}" ]
-    resource('numpy').stage { system "python", "setup.py", *numpy_args }
-    scipy_args = [ "build", "--fcompiler=gnu95",
-                   "install", "--prefix=#{libexec}" ]
-    resource('scipy').stage { system "python", "setup.py", *scipy_args }
-    pyqtgraph_args = [ "setup.py", "install", "--prefix=#{libexec}" ]
-    resource('pyqtgraph').stage { system "python", *pyqtgraph_args }
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    res = %w[numpy scipy pyqtgraph]
+    res += %w[capstone] if build.with? "capstone"
+    res.each do |r|
+      resource(r).stage do
+        system "python", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
 
-    system "autoreconf -f" if build.head?
-    system "./configure", "--prefix=#{prefix}", "--disable-bundles"
-    system "make install"
+    if build.head?
+      ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+      system "python", *Language::Python.setup_install_args(libexec)
+      bin.install Dir["#{libexec}/bin/*"]
+      bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+    else
+      system "./configure", "--prefix=#{prefix}", "--disable-bundles"
+      system "make", "install"
+    end
```

Comment:
Is setup.py missing from the stable release? It'd be nice if we could always use the same build system.